main.go

```diff
@@ -68,10 +68,11 @@ func main() {
 
 	// Create a source.Config from the flags passed by the user.
 	sourceCfg := &source.Config{
-		Namespace:       cfg.Namespace,
-		FQDNTemplate:    cfg.FQDNTemplate,
-		Compatibility:   cfg.Compatibility,
-		PublishInternal: cfg.PublishInternal,
+		Namespace:              cfg.Namespace,
+		FQDNTemplate:           cfg.FQDNTemplate,
+		Compatibility:          cfg.Compatibility,
+		PublishInternal:        cfg.PublishInternal,
+		SourceAnnotationFilter: cfg.SourceAnnotationFilter,
```

Comment:
Let's omit the `Source` prefix because we're already using the `Config` from the source package.